mailgun/route_handlers.py

```diff
@@ -82,6 +75,7 @@ def _handle_recipient(request, recipient):
     body_html = request.POST.get('body-html', '')
     body_plain = request.POST.get('body-plain', '')
     cc_list = address.parse_list(request.POST.get('Cc'))
+    from_ = address.parse(request.POST.get('from'))
```

Comment:
It's the "I want to name a variable X, but X is a reserved word in python, so I have to use X_" syntax.  Which, I'm happily surprised to discover, is actually in pep8, so I'm not just going on what other random people told me.

```
single_trailing_underscore_ : used by convention to avoid conflicts with Python keyword, e.g.

Tkinter.Toplevel(master, class_='ClassName')
```